radiography/src/main/java/radiography/ViewFilters.kt

```diff
@@ -0,0 +1,50 @@
+package radiography
+
+import android.view.View
+
+public object ViewFilters {
+
+  /** A [ViewFilter] that matches everything (does not do any filtering). */
+  @JvmStatic
+  public val NoFilter: ViewFilter = viewFilterFor<Any> { true }
+
+  /**
+   * Filters out root views that don't currently have the window focus from the output of
+   * [Radiography.scan].
+   */
+  @JvmField
+  public val FocusedWindowViewFilter: ViewFilter = viewFilterFor<View> { view ->
+    view.parent?.parent != null || view.hasWindowFocus()
+  }
+
+  /**
+   * Filters out views with ids matching [skippedIds] from the output of [Radiography.scan].
+   */
+  @JvmStatic
+  public fun skipIdsViewFilter(vararg skippedIds: Int): ViewFilter = viewFilterFor<View> { view ->
```

Comment:
Why rename this from `SkipIdsViewFilter` to `skipIdsViewFilter`?

Reply:
Because it's a function now, and the style guidelines dictate it must be normal camel case. The functions in `ViewStateRenderers` follow the same pattern as well.